extensions/pod-menu/package-lock.json

```diff
@@ -1248,6 +1249,12 @@
           "integrity": "sha512-lBN9zLN/oAf68o3zNXYrdCt1kP8WsiGW8Oo2ka41b2IM5JL/S1CTyX1rW0mb/zSuJun0ZUrDxx4sqvYS2FWzPA==",
           "dev": true
         },
+        "typed-emitter": {
```

Comment:
Is it possible to revert these lock changes? if not don't worry.

Reply:
yep, sorry for that. sometimes it happens

Reply:
That it does.